### PR TITLE
Docker: ffmpeg/vtk/pcl installer updates

### PR DESCRIPTION
**Pull request by storypku**

### CODE REVIEW

File: docker/build/installers/install_pcl.sh [hunk 1]
```diff
@@ -31,19 +35,34 @@ if ldconfig -p | grep -q libpcl_common ; then
...
+if [ "${WORKHORSE}" = "cpu" ]; then
+    GPU_OPTIONS="-DWITH_CUDA=OFF"
+fi
+
+info "GPU Options for PCL:\"${GPU_OPTIONS}\""
+
+TARGET_ARCH="$(uname -m)"
+ARCH_OPTIONS=""
+if [ "${TARGET_ARCH}" = "x86_64" ]; then
+    ARCH_OPTIONS="-DPCL_ENABLE_SSE=ON"
```
Comment (changsh726) [inline docker/build/installers/install_pcl.sh:48]: Does this cause compatibility problems between different machines?

Comment (storypku) [reply docker/build/installers/install_pcl.sh:48]: > between

This setting was based on https://src.fedoraproject.org/rpms/pcl/blob/master/f/pcl.spec.
I think the main reason for doing so was, SSE settings were meaningless for non-x86 CPU architectures.

Comment (changsh726) [reply docker/build/installers/install_pcl.sh:48]: But I remember that dreamview module might generate coredump on some machines if SSE is enabled.

Comment (changsh726) [reply docker/build/installers/install_pcl.sh:48]: sorry, I didn't notice that you fixed it by the patch

**Pull request merged**